src/ctia/entity/campaign.clj

```diff
@@ -124,7 +124,12 @@
     :external-id-capabilities :read-campaign
     :can-aggregate?           true
     :histogram-fields         campaign-histogram-fields
-    :enumerable-fields        campaign-enumerable-fields}))
+    :enumerable-fields        campaign-enumerable-fields
+    :searchable-fields        (routes.common/searchable-fields
+                               campaign-fields
+                               :valid_time.start_time
+                               :valid_time.end_time
+                               :activity.date_time)}))
```

Comment:
this is indexed a date, just use fields indexed as text and keyword

Reply:
These are the fields to be ignored. I think I should make that clearer by changing the function's signature, so it is called maybe like this:

```
(searchable-fields {:fields campaign-fields :ignore [:valid_time.start_time :activity.date_time]) 
```

Reply:
https://github.com/threatgrid/ctia/pull/1127/commits/35174d6947e9a11548cc68ea1d54d4264e9fe9c3